<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiffMusic</class>
 <widget class="QWidget" name="DiffMusic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>800</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="perfect_lable">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>90</y>
       <width>181</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcdNumber">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>20</y>
       <width>181</width>
       <height>71</height>
      </rect>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QLabel" name="main_back">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>560</width>
       <height>800</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="HD">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>220</y>
       <width>41</width>
       <height>521</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar {   border: 2px solid grey;   border-radius: 5px;   background-color: yellow;}

QProgressBar {   border: 2px solid grey;   border-radius: 5px;   text-align: center;}

</string>
     </property>
     <property name="value">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QLabel" name="check_label_up">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>650</y>
     <width>140</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="check_label_down">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>650</y>
     <width>140</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="check_label_right">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>650</y>
     <width>140</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="check_label_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>140</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
